easybuild/easyconfigs/s/SSW/SSW-1.1-GCCcore-10.3.0.eb

```diff
@@ -0,0 +1,48 @@
+# Author: Jasper Grimm (UoY)
+
+easyblock = 'MakeCp'
+
+name = 'SSW'
+version = '1.1'
```

Comment:
@smoretti Why not use the latest version 1.2.4? See https://github.com/mengyao/Complete-Striped-Smith-Waterman-Library/releases

Reply:
We already have a patch file for version 1.1. Don't know how many changes are required with the latest release

Reply:
The intention here is to use SSW as dependency for SeqLib.

@smoretti tried to use SSW 1.2.4 as dependency for SeqLib, but that fails horribly due a changed API between SSW 1.1. and 1.2.4:

```
BamRecord.cpp: In constructor SeqLib::BamRecord::BamRecord(const string&, const string&, const string&, const SeqLib::GenomicRegion*):
BamRecord.cpp:151:75: error: no matching function for call to StripedSmithWaterman::Aligner::Align(const char*, const char*, std::__cxx11::basic_string<char>::size_type, StripedSmithWaterman::Filter&, StripedSmithWaterman::Alignment*)
  151 |     aligner.Align(seq.c_str(), ref.c_str(), ref.size(), filter, &alignment);
      |                                                                           ^
In file included from BamRecord.cpp:8:
/user/gent/400/vsc40023/eb_arcaninescratch/RHEL8/skylake-ib/software/SSW/1.2.4-GCCcore-10.3.0/include/ssw/ssw_cpp.h:139:8: note: candidate: bool StripedSmithWaterman::Aligner::Align(const char*, const StripedSmithWaterman::Filter&, StripedSmithWaterman::Alignment*, int32_t) const
  139 |   bool Align(const char* query, const Filter& filter, Alignment* alignment, const int32_t maskLen) const;
      |        ^~~~~
/user/gent/400/vsc40023/eb_arcaninescratch/RHEL8/skylake-ib/software/SSW/1.2.4-GCCcore-10.3.0/include/ssw/ssw_cpp.h:139:8: note:   candidate expects 4 arguments, 5 provided
/user/gent/400/vsc40023/eb_arcaninescratch/RHEL8/skylake-ib/software/SSW/1.2.4-GCCcore-10.3.0/include/ssw/ssw_cpp.h:156:8: note: candidate: bool StripedSmithWaterman::Aligner::Align(const char*, const char*, const int&, const StripedSmithWaterman::Filter&, StripedSmithWaterman::Alignment*, int32_t) const
  156 |   bool Align(const char* query, const char* ref, const int& ref_len,
      |        ^~~~~
/user/gent/400/vsc40023/eb_arcaninescratch/RHEL8/skylake-ib/software/SSW/1.2.4-GCCcore-10.3.0/include/ssw/ssw_cpp.h:156:8: note:   candidate expects 6 arguments, 5 provided
```

The signature of the `Align` function was changed from SSW 1.1 to 1.2.4, see https://github.com/mengyao/Complete-Striped-Smith-Waterman-Library/blob/v1.1/src/ssw_cpp.h#L132
vs https://github.com/mengyao/Complete-Striped-Smith-Waterman-Library/blob/v1.2.4/src/ssw_cpp.h#L139

So it makes sense to stick to SSW 1.1 as dependency for SeqLib (and we should clarify that with a comment in the SeqLib easyconfig where SSW is listed as a dependency)

Reply:
See also https://github.com/mengyao/Complete-Striped-Smith-Waterman-Library/commit/0004b64e1990338caf56f0ecad792feba3f2da78 (hat tip @jfgrimm)